homeworks/Naumkin/HW5/js/first.js

```diff
@@ -0,0 +1,15 @@
+const numbOne = +prompt('Введите первое число');
+const numbTwo = +prompt('Введите второе число');
+
+const showCalcNumber = function () {
+  if (numbOne % 2 === 0 && numbTwo % 2 === 0) {
+    calcNumber = numbOne * numbTwo;
+    return 'Результат вычесления = ' + calcNumber;
```

Comment:
```suggestion
    return 'Результат вычесления = ' + numbOne * numbTwo;
```
Обьявлять переменную таким образом плохо - `calcNumber = numbOne * numbTwo;`